### PR TITLE
mgr/dashboard: load log lines on startup, split out audit log

### PR DESCRIPTION
**Pull request by jcsp**
Previously only saw log lines output after
the module had loaded: now we have `log last`,
let's use it!

Signed-off-by: John Spray <john.spray@redhat.com>

### CODE REVIEW

**Comment by jcsp** [comment]
DNM because haven't tested it yet, just stashing it here for the moment.

File: src/pybind/mgr/dashboard/module.py [hunk 1]
```diff
@@ -377,6 +383,30 @@ def serve(self):
...
+        self.send_command(result, "mon", "", json.dumps({
+            "prefix":"log last",
+            "format": "json"
+            }), "")
+        r, outb, outs = result.wait()
+        if r != 0:
+            # Oh well.  We won't let this stop us though.
+            self.log.error("Error fetching log history (r={0}, \"{1}\")".format(
+                r, outs))
+        else:
```
Comment (tchaikov) [inline src/pybind/mgr/dashboard/module.py]: nit, could early return to reduce the indent level

**Comment by yuriw** [comment]
test this please

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-20_16:21:45-rados-wip-sage-testing2-distro-basic-smithi/

**Pull request merged**